models/TestingInstance.rb

```diff
@@ -66,5 +66,68 @@ def final_result
   def patient_id
     self.resource_references.select{|ref| ref.resource_type == 'Patient'}.first.try(:resource_id)
   end
+  
+  def save_supported_resources(conformance)
+
+    resources = ['Patient', 
```

Comment:
Is there a reason that this resource list does not contain `Condition`, `Device`, or `Immunization`?